pkg/util/traceid.go

```diff
@@ -51,6 +51,14 @@ func TraceIDToHexString(byteID []byte) string {
 	return id
 }
 
+// SpanIDToHexString converts a span ID to its string representation and WITHOUT removing any leading zeros.
+// If the id is < 16, left pad with 0s
+func SpanIDToHexString(byteID []byte) string {
+	id := hex.EncodeToString(byteID)
+	id = strings.TrimLeft(id, "0")
```

Comment:
is this needed?

Reply:
You're right. I was trying to solve for an invalid test case I added.

Reply:
Yes. Without trimming it, padding it 
```
0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x12
```
would return
```
00000000000000000000000000000012
```